<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>911</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnTwist">
    <property name="geometry">
     <rect>
      <x>96</x>
      <y>32</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Twist</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRotation">
    <property name="geometry">
     <rect>
      <x>162</x>
      <y>32</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Rotation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnWidth">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>750</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Width</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnHorizontality">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>750</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Horizontality</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLength">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>750</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Length</string>
    </property>
   </widget>
   <widget class="QSlider" name="sdCut">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>800</y>
      <width>601</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lbState">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>810</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnHome">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>2</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>HOME</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>287</x>
      <y>830</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>CUT MODE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStore">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>30</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Store</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbRatio">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>32</y>
      <width>86</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Basic</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>16:9</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1:1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4:3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Free</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="btnBlur">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>32</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Blur</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelMain">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>630</width>
      <height>630</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnApply">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>30</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Apply</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>642</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCUT_MODE">
    <property name="title">
     <string>CUT MODE</string>
    </property>
   </widget>
   <addaction name="menuCUT_MODE"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
